acp/acp.go

```diff
@@ -89,10 +89,11 @@ type ACP interface {
 	// Otherwise if check failed then an error is returned (and the boolean result should not be used).
 	//
 	// Note(s):
-	// - permission here is a valid DPI permission we are checking for ("read" or "write").
+	// - permissions here is a valid DPI permission we are checking for ("read" or "write").
+	// - returns true if has access to ANY of the given permissions.
 	CheckDocAccess(
 		ctx context.Context,
-		permission DPIPermission,
+		permissions []DPIPermission,
```

Comment:
suggestion: If you move this to the end you could allow simpler calling of it for those who call it with a single item:
```go
CheckDocAccess(
  ctx context.Context,
  docID string,
  permissions ...DPIPermission,
)
```

Reply:
No longer needed, moved the implementation to source hub client